boards/arm/mec172xevb_assy6906/mec172xevb_assy6906.dts

```diff
@@ -26,6 +26,7 @@
 		i2c-0 = &i2c_smb_0;
 		i2c1 = &i2c_smb_1;
 		i2c7 = &i2c_smb_2;
+		eeprom-0 = &eeprom;
```

Comment:
Please list `eeprom` as supported by this board in its YAML file as well.

Reply:
planning to move the eeprom node to the eeprom test application as a board overlay; because eeprom is available only on a certain mec172x variant (mec1727).

Reply:
yes, you are correct, I mentioned incorrectly
EEPROM is available only on MEC1723, updated the comment above